x-pack/plugins/lists/common/schemas/common/schemas.test.ts

```diff
@@ -53,6 +53,15 @@ describe('Common schemas', () => {
       expect(message.schema).toEqual(payload);
     });
 
+    test('it should validate for "wildcard_caseless"', () => {
```

Comment:
I've not added tests like this one in some of the files (`handler` and `mapping`) in this commit cause I think it makes more sense to add tests for validation of path values when we get to it. Else it'll be just copying a lot of tests with a changed operator value.

Reply:
Why did we need to change the List plugin's tests and schema? We don't use their APIs to validate trusted apps. (maybe this schema is used by the ExceptionsListClient to validate payloads?)

Reply:
So this was needed cause the conditions that depend on the operator values use an `EntriesArray` type from the lists plugin. So without this change [routes/trusted_apps/mapping.ts#L76](https://github.com/elastic/kibana/blob/4eb6c22bcceb6e1400e34ff72bcfd2dc821603ac/x-pack/plugins/security_solution/server/endpoint/routes/trusted_apps/mapping.ts#L76) TS complains that the new operator value `wildcard_caseless` never overlaps with `'excluded' | 'included'`.

I could trace this back to [pull/83661](https://github.com/elastic/kibana/pull/83661)

But yes the tests here are not entirely necessary.